src/main/java/com/eerussianguy/firmalife/init/FoodDataFL.java

```diff
@@ -6,6 +6,10 @@
 {
     public static final FoodData CHOCOLATE = new FoodData(4, 0.0F, 0.2F, 0.0F, 0.0F, 0.0F, 0.0F, 0.2F, 1.1F);
     public static final FoodData COCOA_BEANS = new FoodData(4, 0.0F, 0.1F, 0.1F, 0.0F, 0.0F, 0.1F, 0.0F, 0.5F);
+
+    //copied from squash
```

Comment:
would rather you put this in the PR description than as a comment. we're switching to sorta  data generation via spreadsheet for this class soon so comments like this will be deleted

Reply:
meh it's a placeholder anyway.

Reply:
yeah whatever